docker-cronjob/prune_docker_images.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo docker image prune -f
```

Comment:
```suggestion
sudo docker image prune -f

```

i.e. missing newline